src/bios_handler.cpp

```diff
@@ -101,11 +101,10 @@ void IbmBiosHandler::biosAttributesCallback(sdbusplus::message_t& i_msg)
             if (auto l_val = std::get_if<std::string>(
                     &(std::get<5>(std::get<1>(l_attribute)))))
             {
-                (void)l_val; // use when APIs are implemented.
                 std::string l_attributeName = std::get<0>(l_attribute);
                 if (l_attributeName == "hb_memory_mirror_mode")
                 {
-                    // TODO: Save MMM to VPD.
+                    saveAmmToVpd(*l_val);
```

Comment:
this call back will not work - as the BIOS config service name which you have hardcoded in constants.hpp is wrong.

correct service name : xyz.openbmc_project.BIOSConfigManager 

Please correct it.
and btw why this has not been tested?

Reply:
The code has been tested and the call back will work fine.

To register a "propertiesChanged" callback, interface name is required not service name.

So the callback will work fine, its just that the constant I have used in constant.hpp for that value is wrong and should be corrected.

Once we agree on all the open comments. I will make the change and push a PR to change the constant name.

Reply:
Updated PR with correct constant name.

Reply:
@PriyangaRamasamy good catch